src/components/Activity/ActivityItem.js

```diff
@@ -49,7 +44,7 @@ function ActivityItem({ activity }) {
     }
   }, [activity, chainId])
 
-  const canClear = activity.status !== ACTIVITY_STATUS_PENDING
+  const canClear = activity.status !== ActivityStatus.ACTIVITY_STATUS_PENDING
```

Comment:
why are we changing this? it's better to just use the imported status rather than use the `ActivityStatus.` everywhere, probably is because you want to use an enum, but i think that just constants makes it cleaner

Reply:
Thats because we used Symbol and now with typescript we can use enums, thats one of the improvements of typescript.

for example:

```
const X = Symbol('x')
const X1 = Symbol('x1')
const X2 = Symbol('x2')

```
when adding a type to one variable you would need to do,
```
type AAA = {
  variant: X | X1 | X2
}
```

while using enum you can just equal the variant to the enum.
I dont know if I made myself clear :)

Reply:
oh nice! yup thxs!

Reply:
The other major benefit is to type some variable with the Enum type, so enforce you can only receive that Constatns there. Like

```typescript
methodReceiveEnum(status:ActivityStatus)
```